src/signals/settings/categories/components/utils.test.ts

```diff
@@ -18,6 +18,7 @@ const mockFormData: CategoryFormValues = {
   note: 'Test notitie',
   n_days: 5,
   use_calendar_days: 1,
+  icon: null,
```

Comment:
here its the last attribute, but below icon is after name. Isn't it better to keep the same position of attrs

Reply:
Not sure what you mean.

Reply:
I've followed position of attrs of CategoryFormValues in src\signals\settings\categories\types.ts